lib/usage.js

```diff
@@ -247,11 +249,11 @@ module.exports = function (yargs) {
         if (demanded[alias]) yargs.demand(key, demanded[alias].msg)
 
         // type messages.
-        if (~options.boolean.indexOf(alias)) yargs.boolean(key)
-        if (~options.count.indexOf(alias)) yargs.count(key)
-        if (~options.string.indexOf(alias)) yargs.string(key)
-        if (~options.normalize.indexOf(alias)) yargs.normalize(key)
-        if (~options.array.indexOf(alias)) yargs.array(key)
+        if (!options.boolean.indexOf(alias)) yargs.boolean(key)
+        if (!options.count.indexOf(alias)) yargs.count(key)
+        if (!options.string.indexOf(alias)) yargs.string(key)
+        if (!options.normalize.indexOf(alias)) yargs.normalize(key)
+        if (!options.array.indexOf(alias)) yargs.array(key)
```

Comment:
Wrong.  Bitwise NOT !== boolean NOT.  `~` is only falsey if the value is `-1` (ie, the thing is not found).  `!` is falsey if the value is `-1` or > `0` (ie, if the thing is either not found, or found after the first position.)